compliance/queryresultio/src/test/java/org/eclipse/rdf4j/query/resultio/sparqljson/SPARQLJSONTupleBackgroundTest.java

```diff
@@ -82,48 +78,38 @@ public void testBindings1()
 		assertEquals(7, handler.getBindingSets().size());
 
 		// Results are ordered, so first should be book6
-		assertEquals("http://example.org/book/book6",
-				handler.getBindingSets().get(0).getValue("book").stringValue());
+		assertEquals("http://example.org/book/book6", handler.getBindingSets().get(0).getValue("book").stringValue());
 
 		for (BindingSet b : handler.getBindingSets()) {
 			assertNotNull(b.getValue("book"));
 			assertNotNull(b.getValue("title"));
 			assertTrue(b.getValue("book") instanceof IRI);
 			assertTrue(b.getValue("title") instanceof Literal);
 
-			IRI book = (IRI)b.getValue("book");
+			IRI book = (IRI) b.getValue("book");
 			if (book.stringValue().equals("http://example.org/book/book6")) {
 				assertEquals("Harry Potter and the Half-Blood Prince", b.getValue("title").stringValue());
-			}
-			else if (book.stringValue().equals("http://example.org/book/book7")) {
+			} else if (book.stringValue().equals("http://example.org/book/book7")) {
```

Comment:
I like this.

Reply:
indeed, one line of scarce screen real estate saved :-)